src/SFA.DAS.EmployerCommitmentsV2/DependencyResolution/Shared/CommitmentsSharedRegistry.cs

```diff
@@ -0,0 +1,25 @@
+﻿using SFA.DAS.Commitments.Shared.Interfaces;
+using SFA.DAS.Commitments.Shared.Services;
+using SFA.DAS.CommitmentsV2.Api.Client.DependencyResolution;
+using StructureMap;
+
+namespace SFA.DAS.Commitments.Shared.DependencyInjection
+{
+    public class _CommitmentsSharedRegistry : Registry
+    {
+        public _CommitmentsSharedRegistry()
+        {
+            Scan(
+                scan =>
+                {
+                    scan.AssembliesFromApplicationBaseDirectory(a => a.GetName().Name.StartsWith("SFA.DAS.Commitments.Shared"));
+                    scan.RegisterConcreteTypesAgainstTheFirstInterface();
+                });
+
+            For<ICommitmentsService>().Use<CommitmentsService>().Singleton();
+            IncludeRegistry<ApprenticeshipInfoServiceRegistry>();
+            IncludeRegistry<CommitmentsApiClientRegistry>();
+            IncludeRegistry<EncodingRegistry>();
+        }
```

Comment:
Does this class belong in the CommitmentShared Project? The problem with having it here is we need to know how it's internally built

Reply:
Ah, it is in the shared (and being used from there). This class isn't used - have removed